src/index.js

```diff
@@ -24,6 +26,9 @@ const store = createStore(
 
 const history = createHistory();
 
+window.onbeforeunload = (e) => {
+    socketApi.closeSocket();
```

Comment:
Close the socket on windowunload, will have to see how this works out with router, because we don't wan't to close socket on navigation, just refresh or tab close

Reply:
Sounds good, we can try and see once I moved to HashRouter.